src/main/kotlin/no/skatteetaten/aurora/mokey/service/ApplicationDataServiceOpenShift.kt

```diff
@@ -31,29 +31,22 @@ class ApplicationDataServiceOpenShift(
     val addressService: AddressService,
     val imageService: ImageService
 ) : ApplicationDataService {
-    override fun findAllVisibleAffiliations(): List<String> {
-        throw NotImplementedError("findAllVisibleAffiliations is not supported")
-    }
 
     val mtContext = newFixedThreadPoolContext(6, "mokeyPool")
```

Comment:
I am getting a warning for this and when I look at the docs its says:
This API will be replaced in the future.

Reply:
Yes, but there is no alternative now AFAIK so we just leave it in there.